apps/pkey.c

```diff
@@ -186,23 +186,29 @@ int pkey_main(int argc, char **argv)
     if (!noout) {
         if (outformat == FORMAT_PEM) {
             if (pubout) {
-                PEM_write_bio_PUBKEY(out, pkey);
+                if (!PEM_write_bio_PUBKEY(out, pkey))
+                    goto end;
             } else {
                 assert(private);
-                if (traditional)
-                    PEM_write_bio_PrivateKey_traditional(out, pkey, cipher,
+                if (traditional) {
+                    if (!PEM_write_bio_PrivateKey_traditional(out, pkey, cipher,
                                                          NULL, 0, NULL,
```

Comment:
Arguments should be aligned

Reply:
I did not check, but if the lines exceed 80 characters, you might need also to truncate the line earlier by moving the arguments on the following lines.

Reply:
util/openssl-format-source does not complain here.

Reply:
Perfect, I was just being cautious!